sklearn/ensemble/_gb.py

```diff
@@ -883,6 +883,7 @@ class GradientBoostingClassifier(ClassifierMixin, BaseGradientBoosting):
     loss : {'log_loss', 'deviance', 'exponential'}, default='log_loss'
         The loss function to be optimized. 'log_loss' refers to binomial and
         multinomial deviance, the same as used in logistic regression.
+        It contains a factor x2 that have to be neglected to reflect the actual loss.
```

Comment:
I think this is only true for the `log_loss` and `deviance` (which is the same loss).

Reply:
The way to check is to look at how the loss functions are defined in `sklearn/_loss/loss.py`: the loss function classes that have "Half" in their name only compute half of the usual value (for performance reasons when computing the per-sample gradients.

Reply:
Actually not for the GradientBoosting. I think that we still import from `_gb_losses.py`

Reply:
It is as @glemaitre says. The change to the loss function module was blocked by a deprecation period for the `loss_` attribute of `GradientBoostingRegressor`. This is now over (1.3).